go-controller/pkg/libovsdbops/portgroup.go

```diff
@@ -37,10 +37,14 @@ func CreateOrUpdatePortGroupsOps(nbClient libovsdbclient.Client, ops []libovsdb.
 	for i := range pgs {
 		pg := pgs[i]
 		opModel := operationModel{
-			Model:          pg,
-			OnModelUpdates: onModelUpdatesAll(),
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model: pg,
+			OnModelUpdates: []interface{}{
+				&pg.ACLs,
+				&pg.Ports,
+				&pg.ExternalIDs,
```

Comment:
Ok, I think we should then take care to have all fields we could ever be interesting in updating in this list.

I think it would fit to have a `getAllUpdateablePortGroupFields` method in `model.go` (where we are putting stuff we hope to be able to generate one day) with this field list.

Also maybe update the function doc text to explicitly state all fields are updated?